test/functional/validation-options.spec.ts

```diff
@@ -20,1043 +12,1034 @@ const validator = new Validator();
 // Specifications: common decorators
 // -------------------------------------------------------------------------
 
-describe("validation options", function() {
```

Comment:
Why did you remove `describe`? I saw you remove root describe in some specs (but not at all of them). I would love to keep root describe.

Reply:
Jest does not require root describe blocks unless you wish to include before/after setup for all tests within the block. As it was superfluous I removed it.